setup.py

```diff
@@ -0,0 +1,63 @@
+"""A setuptools based setup module.
```

Comment:
`setup.py` can be deleted, since `itertools` is available in CPython and won't need to be deployed to PyPI.

Reply:
Done